script/tools/linux-install-clang

```diff
@@ -42,9 +42,9 @@ function install_clang() {
   echo 'Installing clang'
   clear_apt \
   && ${SUDO} apt-get -qqy install {clang,clang-format}-"${CLANG_VERSION}" \
-  && ${SUDO} cp -f /usr/bin/clang-3.8 /usr/bin/clang \
-  && ${SUDO} cp -f /usr/bin/clang++-3.8 /usr/bin/clang++ \
-  && ${SUDO} cp -f /usr/bin/clang-format-3.8 /usr/bin/clang-format \
+  && ${SUDO} cp -f "/usr/bin/clang-${CLANG_VERSION}" /usr/bin/clang \
```

Comment:
For c++ code format,  we have to stay at 3.8 version.   Not sure if > 3.8 version generate same format or not?

Reply:
It should, CLANG_VERSION is set at 3.8 at line 39